data/outreachy.yml

```diff
@@ -276,4 +276,19 @@ rounds:
         - Reynir Björnsson
         video: https://watch.ocaml.org/w/kTwq4AVQtRtFuQx9R5cPro
 
-       
+  - name: Summer 2023
+    projects:
```

Comment:
There is already a "Summer 2023" entry. This project should be added to the list, instead of creating another outreachy round.
```suggestion
```

Reply:
I'll fix this asap